projects/skia/build.sh

```diff
@@ -38,7 +38,7 @@ else
   CFLAGS= CXXFLAGS="-stdlib=libc++" cmake .. -D$CMAKE_SANITIZER=1
 fi
 
-make -j2 libGLESv2 libEGL
+make -j14 libGLESv2 libEGL
```

Comment:
Why this change.

Reply:
I suggested this change to make the compilations work faster locally for Zepeng. Do you think this will cause a problem in other places?

Reply:
ok fine then! i think it should be ok on builder.

Reply:
I'm trying to replace this with something that will intelligently scale at https://github.com/google/oss-fuzz/pull/3979 .